docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
+<img src="images/EditRefrence1.png" width="600"/>
```

Comment:
<img width="178" alt="ss5" src="https://user-images.githubusercontent.com/60163283/140518191-93dbb177-c469-4c4f-a767-73e812a45b1d.PNG">
Method does not exist its instruments.get which is an arraylist method so can ignore.